x-pack/legacy/plugins/ml/server/routes/calendars.js

```diff
@@ -53,12 +59,17 @@ export function calendars({ commonRouteConfig, elasticsearchPlugin, route }) {
 
   route({
     method: 'GET',
-    path: '/api/ml/calendars/{calendarId}',
+    path: '/api/ml/calendars/{calendarIds}',
     handler(request) {
       const callWithRequest = callWithRequestFactory(elasticsearchPlugin, request);
-      const calendarId = request.params.calendarId;
-      return getCalendar(callWithRequest, calendarId)
-        .catch(resp => wrapError(resp));
+      const calendarIds = request.params.calendarIds.split(',');
+      if (calendarIds.leading === 1) {
```

Comment:
is this supposed to be `length`?

Reply:
wow good catch, autocomplete 😬

Reply:
https://github.com/elastic/kibana/pull/51281/commits/b153888e4c30490dfcbf2b7f0299eb54daea65d9